src/coprocessor/dag/executor/mod.rs

```diff
@@ -51,6 +51,7 @@ pub use self::selection::SelectionExecutor;
 pub use self::table_scan::TableScanExecutor;
 pub use self::topn::TopNExecutor;
 
+/// An expression tree visitor to extract all column offsets in the tree.
```

Comment:
@Hijiao I agree with you. It would be clearer using either of the following two versions:

- `An expression tree visitor that extracts ...`
- `An expression tree visitor used to extract ...`